apps/devtools/app/scripts/atoms/devtool.ts

```diff
@@ -0,0 +1,4 @@
+import { atom } from 'jotai';
+
+export const recordingAtAtom = atom<number>(0);
+export const isRecordingAtom = atom<boolean>(get => !!get(recordingAtAtom));
```

Comment:
Haha you don't waste time. It would be really awesome to get a hands-on crash course on jotai after you've used it a little. Seems like you have a good grasp already.

Reply:
Started this before digging into it this week, lots of stuff we can do with this tech. This was actually a simple use case to try it out.